src/Evp/Notification/Entity/OperationNotification.php

```diff
@@ -639,6 +661,7 @@ public static function fromParams(array $params)
             ->setReferenceToBeneficiary(isset($params['reference_to_beneficiary']) ? $params['reference_to_beneficiary'] : null)
             ->setReferenceToPayer(isset($params['reference_to_payer']) ? $params['reference_to_payer'] : null)
             ->setStatementId(isset($params['statement_id']) ? $params['statement_id'] : null)
+            ->setCreatedAt(isset($params['created_at']) ? $params['created_at'] : null)
```

Comment:
Let's use DateTime as it represents the data more clearly.
To avoid timezone issues, we use this type of construction:
```
$createdAt = new \DateTime();
$createdAt->setTimestamp($timestamp);
```

Constructing with `@` in constructor forces UTC timezone to be used on resulting object.